### PR TITLE
Task-48155: Cancel button is badly displayed in Confirmation popup

### PR DESCRIPTION
**Pull request by lbengayed**
Problem: the cancel popup button label description didn't exist in the .properties file and the css hover button properties.
How it was solved: use the cancel label that exit in .properties and fix css hover problem.

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you rebase your PR in order to have only your commit ?
Thanks

**Comment by lbengayed** [comment]
> Can you rebase your PR in order to have only your commit ?
> Thanks

done!

File: apps/portlet-documents/src/main/webapp/vue-app/legacy-composer-attachments/components/ExoDropdownMenu.vue [hunk 1]
```diff
@@ -8,16 +8,18 @@
     @blur="closeMenu">
     <li
       :class="!selectedFolder.canRemove ? 'disabled' : ''"
-      class="folderActionsMenuItem btn delete"
+      class="folderActionsMenuItem  delete"
```
Comment (rdenarie) [inline apps/portlet-documents/src/main/webapp/vue-app/legacy-composer-attachments/components/ExoDropdownMenu.vue]:
```suggestion
      class="folderActionsMenuItem delete"
```

File: apps/portlet-documents/src/main/webapp/vue-app/legacy-composer-attachments/components/ExoDropdownMenu.vue [hunk 2]
```diff
@@ -8,16 +8,18 @@
...
       @click="deleteFolder()">
-      <i class="uiIconTrash"></i>{{ $t('attachments.filesFoldersSelector.action.delete') }}
+      <i class="uiIconTrash"></i>
+      <span>{{ $t('attachments.filesFoldersSelector.action.delete') }}</span>
     </li>
     <li
       :class="!selectedFolder.canRemove ? 'disabled' : ''"
-      class="folderActionsMenuItem btn rename"
+      class="folderActionsMenuItem  rename"
```
Comment (rdenarie) [inline apps/portlet-documents/src/main/webapp/vue-app/legacy-composer-attachments/components/ExoDropdownMenu.vue]:
```suggestion
      class="folderActionsMenuItem rename"
```

File: apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less [hunk 3]
```diff
@@ -911,14 +911,11 @@
...
-
-                &:hover {
-                  background: #578dc9;
-                  color: #fff;
-
-                  i {
-                    color: #fff;
+                &:hover{
+                  i,span {
+                    color: #ffffff!important;
```
Comment (rdenarie) [inline apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less]: Does important is necessary here ?

File: apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less [hunk 4]
```diff
@@ -911,14 +911,11 @@
...
                   }
+                  background-image: linear-gradient(to bottom, @primaryColor, @primaryColor)!important;
```
Comment (rdenarie) [inline apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less]: Does important is necessary here ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less (hunk 3 above)
Comment (lbengayed) [reply apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less]: done!

File: apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less (hunk 4 above)
Comment (lbengayed) [reply apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/legacyComposerAttachments.less]: done!

**Review by rdenarie — APPROVED** [review]

**Pull request merged**